### PR TITLE
Milestone -5

### PR DESCRIPTION
**Pull request by tigamadou**
- App component converted to stateful component
- Added total, next, and operation to App component
- Implemented render function that should display Display and ButtonPanel
- Implemented the handleClick function
- Passed the total or next value to Display component
- Passed handleClick as a clickHandler property to ButtonPanel
- Implemented the handleClick function in ButtonPanel component
- Passed the clickHandler property to each button in the buttonPanel component
- Implement the handleClick function in the button component

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -1,17 +1,35 @@
-import React from 'react';
+import React, { Component } from 'react';
 
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
 // eslint-disable-next-line no-unused-vars
```
Comment (Torres-ssf) [inline src/components/App.js]: - [x] We can safely remove this `eslint` disable comment here, it will not make a difference in your app.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes Required :muscle:
Hi @tigamadou :hand: 
Awesome job on this milestone so far :clap: 
It seems all calculation operations are working as they should be. Congratulations :tada: 
But we need to make some changes before moving on.

- [x] I got an error while trying to run your application locally. The `babel-eslint` dependency is missing, we need to add it as a dev dependency. You can check [here](https://github.com/microverseinc/linters-config/tree/master/react-redux) all dependencies that we need to add to our repo to properly set up both `eslint` and `stylelint`.
Screenshot from 2021-04-08 08-39-36

- [x] The calculator display should show the 0 value when no value was inputted yet. This is why at the Display component we gave a default value for the result props.
bug1

- [x] After selecting an operator, the calculator should show the next digits that are being inputted by the user, just like an actual calculator
bug2

- [x] We can safely delete the `logo.svg` file since it's not being used in your App.

- [x] Let's remove the commented line at the linters.yml file.
https://github.com/tigamadou/calculator/blob/faab53ad59f3619a4af233fa2582c51c17f0ad9f/.github/workflows/linters.yml#L35-L37

Happy coding :computer: 
**Sérgio Torres**

File: src/components/App.js (hunk 1 above)
Comment (tigamadou) [reply src/components/App.js]: @Torres-ssf Thank you very much. 👍

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @tigamadou :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Comment by tigamadou** [comment]
@MuriloRoque Thanks a lot. 👍

**Pull request merged**